legacy/opentracing-extension/src/main/java/org/wildfly/extension/microprofile/opentracing/SubsystemDefinition.java

```diff
@@ -106,24 +77,20 @@ public void registerChildren(ManagementResourceRegistration resourceRegistration
     }
 
     @Override
-    public Collection<AttributeDefinition> getAttributes() {
-        return Collections.singletonList(DEFAULT_TRACER);
+    public void registerOperations(ManagementResourceRegistration resourceRegistration) {
+        super.registerOperations(resourceRegistration);
+        MigrateOperation.registerOperations(resourceRegistration, getResourceDescriptionResolver());
     }
 
+//    public Collection<AttributeDefinition> getAttributes() {
+//        return Collections.singletonList(DEFAULT_TRACER);
+//    }
+
     @Override
     public void registerAttributes(ManagementResourceRegistration resourceRegistration) {
         resourceRegistration.registerReadWriteAttribute(DEFAULT_TRACER, null, new DefaultTracerWriteAttributeHandler());
```

Comment:
There are a couple options:

1) Pass DEFAULT_TRACER as a param to the superclass constructor and remove this method entirely. Or
2) s/DefaultTracerWriteAttributeHandler()/new new ModelOnlyWriteAttributeHandler(DEFAULT_TRACER)/g

Either way the DefaultTracerWriteAttributeHandler class should be removed.

This is all ok because DEFAULT_TRACER_CAPABILITY is a non-longer-used internal detail. It has no function on a Domain Controller managing a legacy host so the code to register/unregister it can be dropped.

Reply:
Fixed.